src/tools/crossgen2/crossgen2/Program.cs

```diff
@@ -135,6 +136,7 @@ private ArgumentSyntax ParseCommandLine(string[] args)
                 syntax.DefineOption("systemmodule", ref _systemModuleName, "System module name (default: System.Private.CoreLib)");
                 syntax.DefineOption("waitfordebugger", ref waitForDebugger, "Pause to give opportunity to attach debugger");
                 syntax.DefineOptionList("codegenopt", ref _codegenOptions, "Define a codegen option");
+                syntax.DefineOption("doNotFailOnCodegenCompilationExceptions", ref _doNotFailOnCodegenCompilationExceptions, "Disable behavior where unexpected compilation failures cause overall compilation failure");
```

Comment:
Can we make the name shorter and match casing of the other switches? How about `--ignorecompilefailures` or `--resilient`?

Reply:
I like --resilient